src/Http/Controllers/Api/Geojson/ImageAnnotationsController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace Biigle\Modules\Geo\Http\Controllers\Api\Geojson;
+
+use DB;
+use Biigle\Image;
+use GeoJson\Feature\{Feature, FeatureCollection};
+use GeoJson\Geometry\Point;
+use Biigle\Http\Controllers\Api\Controller;
+use League\Flysystem\FileNotFoundException;
+
+class ImageAnnotationsController extends Controller{
+
+  public function index($id)
+  {
+    $image = Image::findOrFail($id);
+    $this->authorize('access', $image);
+
+    $metadata = $image->metadata;
+    $image_width_m = 2 * (float)$image->metadata['distance_to_ground'];
+    $result = $image->annotations->map(function($annotation) use($image, $metadata,$image_width_m){
+      $annotation_points = array_slice($annotation->points, 0, 2);
+      $image_center = [($image->width)/2, ($image->height)/2];
+
+      # Finding Annotation Point with respect to center of the image.
+      $annotation_point = [($annotation_points[0] - $image_center[0]), ($image_center[1] - $annotation_points[1])];
+
+      # X and Y Coordinate Rotations According to Yaw
+      $rotated_X = $annotation_point[0] * cos((Float)$metadata['yaw']) + $annotation_point[1] * sin((Float)$metadata['yaw']);
+      $rotated_Y = $annotation_point[0] * sin((Float)$metadata['yaw']) + $annotation_point[1] * cos((Float)$metadata['yaw']);
+
+      $scaling_factor = $image_width_m/$image->width;
+
+      # Coordinate Offset in Meters
+      $coordinate_offset_meters = array_map(function($point) use($scaling_factor){return $point * $scaling_factor;}, [$rotated_X, $rotated_Y]);
```

Comment:
Please use multiple lines for inline functions.
```suggestion
      $coordinate_offset_meters = array_map(function ($point) use ($scaling_factor) {
         return $point * $scaling_factor;
      }, [$rotated_X, $rotated_Y]);
```